distribution/xfer/download.go

```diff
@@ -18,6 +18,8 @@ import (
 
 const maxDownloadAttempts = 5
 
+var waitDuration = time.Second
```

Comment:
Not sure this is the right name for it, and it needs a comment too :sweat_smile: